rs-client-demo/src/main.rs

```diff
@@ -15,6 +16,14 @@ impl Verification for CustomVerifier {
     }
 }
 
+fn print_records(records: Vec<serde_json::Value>) {
+    for record in &records {
+        println!("-------------");
```

Comment:
This will give double lines between records.  You are explicitly quoting `\n`, right, so there are no embedded newlines?  Just print each record on a line.

Reply:
done